src/NServiceBus.Metrics.Tests/RawData/RingBufferTests.cs

```diff
@@ -66,6 +66,37 @@ public void Full_write_multiple_times()
         }
 
         [Test]
+        public void Estimate_just_initialized_ring_size()
```

Comment:
There seems to be no test for a ring buffer with something in it in between empty and full.